l10n_ro_partner_unique/models/res_partner.py

```diff
@@ -10,22 +10,20 @@ class ResPartner(models.Model):
     _inherit = "res.partner"
 
     @api.model
-    def _get_vat_nrc_constrain_domain(self):
+    def _get_vat_constrain_domain(self):
         domain = [
             ("parent_id", "=", False),
             ("vat", "=", self.vat),
-            ("nrc", "=", self.nrc),
         ]
```

Comment:
Ar trebui sa lasam si nrc, dar in domeniul sa verificam daca este egal sau este false

Reply:
Plus ar trebui sa lasam si metoda pentru vat nrc, iar in constrains sa avem separat daca e completat si nrc, deci daca vat si nrc sunt completate  utilizam domeniu initial:  
def _get_vat_nrc_constrain_domain(self):
        domain = [
            ("parent_id", "=", False),
            ("vat", "=", self.vat),
            ("nrc", "=", self.nrc),	
        ], daca are completat doar vat
def _get_vat_constrain_domain(self):
        domain = [
            ("parent_id", "=", False),
            ("vat", "=", self.vat),
            '|', ("nrc", "=", self.nrc), ('nrc', '=', False)	
        ],

Reply:
daca partenerii sunt unici dupa cui nu mai are sens a faci si verificarea dupa nrc.